ui/app/AppLayouts/Wallet/components/SendModalContent.qml

```diff
@@ -32,30 +32,37 @@ Item {
         sendingSuccess.open()
     }
 
-    function validate() {
-        const isRecipientValid = selectRecipient.validate()
-        const isAssetAndAmountValid = txtAmount.validate()
-        if (txtPassword.text === "") {
+    function validatePassword() {
+        if (transactionSigner.passwordInput.text === "") {
             //% "You need to enter a password"
             passwordValidationError = qsTrId("you-need-to-enter-a-password")
-        } else if (txtPassword.text.length < 4) {
+        } else if (transactionSigner.passwordInput.text.length < 4) {
             //% "Password needs to be 4 characters or more"
             passwordValidationError = qsTrId("password-needs-to-be-4-characters-or-more")
         } else {
             passwordValidationError = ""
         }
+        
+        return passwordValidationError === "" 
+    }
```

Comment:
I've extracted the password validation into its own function for now so we can trigger separately from the general validation. This will most likely be revised in the new send transaction modal